aida-x/loong.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index dd6a5b8..a10c58b 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -4,7 +4,7 @@ pkgbase=aida-x
+ _name="${pkgbase^^}"
+ pkgname=(
+   aida-x
+-  aida-x-{clap,lv2,standalone,vst,vst3}
++  aida-x-{clap,lv2,standalone,vst}
+ )
+ pkgver=1.1.0
+ pkgrel=1
+@@ -38,6 +38,11 @@ _pick() {
+   done
+ }
+ 
++prepare() {
++    cd AIDA-X-$pkgver
++    patch -p1 -i ../disable-vst3.patch
++}
++
+ build() {
+   local cmake_options=(
+     -B build
+@@ -155,6 +160,7 @@ package_aida-x-vst() {
+   install -vDm 755 build/bin/$_name-vst2.so -t "$pkgdir/usr/lib/vst/"
+ }
+ 
++: <<COMMENT_SEPARATOR
```

Comment:
If you remove `vst3` from `pkgname` array, there is no need to comment out the corresponding package function.